### PR TITLE
npscorer.py: Py3 compat and importable from other locations

### PR DESCRIPTION
**Pull request by apahl**
* made Python3 compatible
* modified load path for the publicnp.model file to enable import of the module from other locations
* my editor removed some trailing whitespace

### CODE REVIEW

File: Contrib/NP_Score/npscorer.py [hunk 1]
```diff
@@ -13,11 +13,13 @@
 # peter ertl, august 2015
 #
 
+from __future__ import print_function
 from rdkit import Chem
 from rdkit.Chem import rdMolDescriptors
 import sys,math,gzip,pickle
+import os.path as op
```
Comment (greglandrum) [inline Contrib/NP_Score/npscorer.py]: I'm not a big fan of this kind of aliasing. It doesn't save you that much typing (particularly here), but it does make the code harder to read.

File: Contrib/NP_Score/npscorer.py [hunk 2]
```diff
@@ -36,20 +38,20 @@ def scoreMol(mol,fscore):
...
-  if score > 4: 
+  if score > 4:
     score = 4. + math.log10(score - 4. + 1.)
   if score < -4:
     score = -4. - math.log10(-4. -score + 1.)
   return score
-  
-def processMols(fscore,suppl,outf):
+
+def processMols(fscore,suppl):
```
Comment (greglandrum) [inline Contrib/NP_Score/npscorer.py]: nice catch. It's a bit of a mystery why this managed to survive so long.

File: Contrib/NP_Score/npscorer.py (hunk 1 above)
Comment (apahl) [inline Contrib/NP_Score/npscorer.py]: That is no problem, I just do that out of habit. Should I retract the PR and submit a new one?

File: Contrib/NP_Score/npscorer.py (hunk 2 above)
Comment (bp-kelley) [inline Contrib/NP_Score/npscorer.py]: outf was passed in as None, so it wasn't really doing anything :)

**Comment by bp-kelley** [comment]
I'm going to merge this, but perhaps we should perhaps have a testing suite for contrib?  There is enough useful code here it might be nice to know if we broke anything.

**Pull request merged**